src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form-big"
+    >
+      <input
+        type="text"
+        class="input input-big"
+        data-qa="big keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <div class="search search-big"></div>
+    </form>
+
+    <form
+      action="#"
+      method="post"
+      class="form-small"
+    >
+      <input
+        type="text"
+        id="input-small"
+        class="input input-small"
+        data-qa="small"
```

Comment:
read the task attentively
>add attribute data-qa="big" for big search form, and data-qa="small" for small
add attribute data-qa="keypress" to input in big search form


```suggestion
    <form
      action="#"
      method="post"
      class="form-small"
      data-qa="small"
    >
      <input
        type="text"
        id="input-small"
        class="input input-small"
```